textproc/dblatex/Portfile

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
Is there any particular reason for changing the modeline (other than perhaps vim not respecting the setting)? If you want some changes in this line, it makes sense to ask on the macports-devel mailing list rather than deliberately changing a single `Portfile` to be inconsistent with the rest.

Reply:
Well, it avoids an overlong line, also separating vim and Emacs settings (which are thus easier to comprehend IMHO).  Additionally, there is at least one other Portfile (`python/py-cdb/Portfile`) that also uses an Emacs local variables block at the end of the file – I thus assume that it is an allowed (I dare to say, better) variant.  I *did* check :-)

Reply:
Still, please bring this discussion up to the developer mailing list. I don't feel competent enough to judge about it. But I feel that we should strive to keep the design consistent, and either change all modelines to what is considered a better solution, or none.

Reply:
The guide says we have a standard modeline now. It should be left as-is. The fact that one port is inconsistent doesn't matter.